documentation/asciidoc/computers/raspberry-pi/boot-nvme.adoc

```diff
@@ -1,35 +1,76 @@
 == NVMe SSD boot
 
-NVMe (non-volatile memory express) is a standard for accessing solid state drives (SSDs) via a PCIe bus. You can connect these drives via the PCIe slot on a Compute Module 4 (CM4) IO board, allowing a CM4 to boot from SSD.
+NVMe (Non-Volatile Memory express) is a standard for external storage access over a PCIe bus. You can connect NVMe drives via the PCIe slot on a Compute Module 4 (CM4) IO board or Raspberry Pi 5. With some additional configuration, you can boot from an NVMe drive.
 
-=== Required hardware
+=== Prerequisites
 
-You need an NVMe M.2 SSD. You cannot plug an M.2 SSD directly into the PCIe slot on the IO board; an adapter is needed. Be careful to get the correct type: a suitable adapter can be found online by searching for "PCI-E 3.0 ×1 lane to M.2 NGFF M-Key SSD NVMe PCI Express adapter card".
+==== Hardware
 
-The latest version of Raspberry Pi OS supports booting from NVMe drives. To check that your NVMe drive is connected correctly, boot Raspberry Pi OS from another drive and run `ls -l /dev/nvme*`. Example output is shown below.
+* NVMe M.2 SSD
+* an adapter to convert from PCIe FFC to an M.2 standard.
```

Comment:
Minor nitpick, but whilst the Pi5 has a PCIe FFC, the CM4IO board has a regular PCIe slot.